infrastructure/identity-provider/conf-files/keycloak-configuration.yaml

```diff
@@ -1,325 +1,555 @@
-  extraArgs: ""
-  ## Username for the initial Keycloak admin user
-  username: keycloak
-  ## Password for the initial Keycloak admin user. Applicable only if existingSecret is not set.
-  ## If not set, a random 10 characters password will be used
-  password: ""
-  # Specifies an existing secret to be used for the admin password
-  existingSecret: ""
-  # The key in the existing secret that stores the password
-  existingSecretKey: password
-  ## jGroups configuration (only for HA deployment)
-  jgroups:
-    discoveryProtocol: dns.DNS_PING
-    discoveryProperties: >
-      "dns_query={{ template "keycloak.fullname" . }}-headless.{{ .Release.Namespace
-      }}.svc.{{ .Values.clusterDomain }}"
-
-  ## Allows the specification of additional environment variables for Keycloak
-  extraEnv: |
-    - name: PROXY_ADDRESS_FORWARDING
-      value: "true"
-    - name: KEYCLOAK_STATISTICS
-      value: "db,http"
-    #- name: WILDFLY_LOGLEVEL
-    #  value: DEBUG
-    # - name: CACHE_OWNERS
-    #   value: "2"
-    # - name: DB_QUERY_TIMEOUT
-    #   value: "60"
-    # - name: DB_VALIDATE_ON_MATCH
-    #   value: true
-    # - name: DB_USE_CAST_FAIL
-    #   value: false
-  affinity: |
-    podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
+
+## Overrides the default entrypoint of the Keycloak container
+command: []
+
+## Overrides the default args for the Keycloak container
+args: []
+
+# Additional environment variables for Keycloak
+extraEnv: |
+  - name: JAVA_OPTS
+    value: >-
+      -XX:+UseContainerSupport
+      -XX:MaxRAMPercentage=50.0
+      -Djava.net.preferIPv4Stack=true
+      -Djboss.modules.system.pkgs=$JBOSS_MODULES_SYSTEM_PKGS
+      -Djava.awt.headless=true
+  - name: PROXY_ADDRESS_FORWARDING
+    value: "true"
+  - name: KEYCLOAK_STATISTICS
+    value: "db,http"
+  - name: JGROUPS_DISCOVERY_PROTOCOL
+    value: kubernetes.KUBE_PING
+  - name: KUBERNETES_NAMESPACE
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: metadata.namespace
+  - name: CACHE_OWNERS_COUNT
+    value: "2"
+  - name: CACHE_OWNERS_AUTH_SESSIONS_COUNT
+    value: "2"
+  - name: DB_VENDOR
+    value: postgres
+  - name: DB_ADDR
+    value: keycloak-db-cluster
+  - name: DB_PORT
+    value: "5432"
+  - name: DB_DATABASE
+    value: keycloak
+  - name: DB_USER_FILE
+    value: /secrets/db-creds/username
+  - name: DB_PASSWORD_FILE
+    value: /secrets/db-creds/password
+
+# Additional environment variables for Keycloak mapped from Secret or ConfigMap
+extraEnvFrom: ""
+
+#  Pod priority class name
+priorityClassName: ""
+
+# Pod affinity
+affinity: |
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchLabels:
+            {{- include "keycloak.selectorLabels" . | nindent 10 }}
+          matchExpressions:
+            - key: app.kubernetes.io/component
+              operator: NotIn
+              values:
+                - test
```

Comment:
Should be configured with the correct label, otherwise it is useless. The `preferredDuringSchedulingIgnoredDuringExecution` term below should be commented, since we have only one zone